app/src/main/java/com/sopt/gongbaek/presentation/ui/grouplist/component/LoadingViewAnimation.kt

```diff
@@ -0,0 +1,83 @@
+package com.sopt.gongbaek.presentation.ui.grouplist.component
+
+import androidx.compose.animation.core.FastOutSlowInEasing
+import androidx.compose.animation.core.animateFloatAsState
+import androidx.compose.animation.core.tween
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.offset
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableIntStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import com.sopt.gongbaek.R
+import com.sopt.gongbaek.ui.theme.GongBaekTheme
+import kotlinx.coroutines.delay
+
+@Composable
+fun LoadingViewAnimation() {
+    val steps = remember { mutableListOf(false, false, false) }
+    val activeStep = remember { mutableIntStateOf(0) }
+
+    LaunchedEffect(Unit) {
+        while (true) {
+            for (i in steps.indices) {
+                activeStep.intValue = i
+                delay(300L)
+            }
+        }
+    }
```

Comment:
[2] 단순 범위를 사용하기위한 용도라면 steps 리스트 선언 없이 repeat을 사용해보는건 어떤지 의견 남겨봅니다!
```suggestion
    val activeStep = remember { mutableIntStateOf(0) }
    
    LaunchedEffect(Unit) {
        while (true) {
            repeat(3) {
                activeStep.intValue = it
                delay(300L)
            }
        }
    }
```